### PR TITLE
#trivial MOBILE-961 Add enable/disable fields behavior

### PR DESCRIPTION
**Pull request by mclomello**
@victorg1991 @dgarciasarai

### CODE REVIEW

File: android/library/src/main/java/com/liferay/mobile/screens/viewsets/defaultviews/ddm/form/fields/DDMFieldCheckboxMultipleView.java [hunk 1]
```diff
@@ -165,7 +165,9 @@ public void setParentView(View view) {
     public void setUpdateMode(boolean enabled) {
         if (this.field.isShowLabel()) {
             TextView label = findViewById(R.id.liferay_ddm_label);
-            label.setEnabled(enabled);
+	        if (label != null) {
+		        label.setEnabled(enabled);
+	        }
```
Comment (victorg1991) [inline android/library/src/main/java/com/liferay/mobile/screens/viewsets/defaultviews/ddm/form/fields/DDMFieldCheckboxMultipleView.java]: Can we extract this code to a utility class? It is repeated in several places

Comment (mclomello) [reply android/library/src/main/java/com/liferay/mobile/screens/viewsets/defaultviews/ddm/form/fields/DDMFieldCheckboxMultipleView.java]: @victorg1991 Yes, we can extract this for a util class. I made that change right now. Thanks.

**Comment by victorg1991** [comment]
Is the project compiling in your machine? Travis is failing at compile it

**Comment by mclomello** [comment]
@victorg1991 Yeah, the project is compiling correctly. The failure on Travis is related for iOS, but I don't made any change in this package.

**Comment by victorg1991** [comment]
Just started reviewing :)

**Comment by phcp** [comment]
@mclomello please check the Travis log, Android project is not building. 
https://travis-ci.org/liferay/liferay-screens/jobs/476519818#L1583

**Comment by victorg1991** [comment]
I've removed some imports that weren't working, let's wait for Travis now :)

**Comment by mclomello** [comment]
@phcp Yeah, sorry i didn't see these imports. Thank you.
@victorg1991 Thanks for fix it.

**Comment by victorg1991** [comment]
@mclomello Niice :D thanks!

**Pull request merged**